metrics/mir_team_bugs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+"""Submit metrics for MIR Team queue sizes.
+
+Copyright 2019 Canonical Ltd.
+Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
+
+Based on triage.py:
+Joshua Powers <josh.powers@canonical.com>
+"""
+import argparse
+
+from metrics.helpers import lp
+from metrics.helpers import util
+
+
+def collect(dryrun=False):
+    """Submit data to Push Gateway."""
+    unassigned = lp.get_team_unassigned_bugs(team='ubuntu-mir',
+                                             distro='Ubuntu')
+    incomplete = lp.get_team_incomplete_bugs(team='ubuntu-mir',
+                                             distro='Ubuntu')
+    pending = lp.get_mir_in_review()
+    security = lp.get_mir_in_security_review()
+    approved = lp.get_approved_mirs()
+
+    print('Unassigned Total: %s' % unassigned)
+    print('Incomplete Total: %s' % incomplete)
+    print('Pending Total: %s' % pending)
+    print('Security Total: %s' % security)
+    print('Approved Total: %s' % approved)
+
+    if not dryrun:
+        print('Pushing data...')
+
+        data = [
+            {
+                'measurement': 'mir_team_bugs',
```

Comment:
I think we'd want to have the measurement start with "distro_" so the filter list is grouped in a sensible way.